.gitignore

```diff
@@ -17,6 +17,7 @@ http_service/models/*model*
 
 data/
 sheets/
+git_repo-shared/
```

Comment:
Unrelated change!

Reply:
The folder was created when I ran 

scripts.commit_classifier regressor repo_dir --revision REVISION 

I will remove it